app/assets/stylesheets/config/_variables.scss

```diff
@@ -124,6 +124,7 @@
 
   --z-negative: -1; // to hide something behind everything else
   --z-elevate: 1; // to elevate something, just a little, in relation to its sibling
+  --z-button: 10; // button containers in liquid tag embeds
```

Comment:
This value certainly works for fixing the bug; however, I would like Design's input on the choice of `10` as the z-index. @ludwiczakpawel

Reply:
Thanks for pinging, @msarit - great catch!

This indeed isn't necessarily the direction we wanna go with. There's clearly an issue with that button z-index but introducing another variable for that isn't the idea solution here. What we want instead is replace that `100` value in few liquid tags-related CSS files with another variable that already exists: `--z-elevate` - this is just enough to make sure that the button is ABOVE that "fade out gradient" (if we can call it that way) within the context of liquid tag.

And so we don't need to introduce another variable just for this one thing :).

Reply:
@ludwiczakpawel Thanks for your feedback. Just updated the PR as you suggested.